packages/expo-updates/ios/EXUpdates/Update/RawManifest/EXUpdatesRawManifest.h

```diff
@@ -55,6 +55,9 @@ NS_ASSUME_NONNULL_BEGIN
 - (nullable NSString *)orientation;
 - (nullable NSDictionary *)experiments;
 - (nullable NSDictionary *)developer;
+- (nullable NSString *)facebookAppId;
+- (nullable NSString *)facebookApplicationName;
```

Comment:
There might be some question why this is Application name and the field above is App ID

Reply:
Yeah, at this point it's just to be consistent with android.